src/components/Admin/CheckIn.jsx

```diff
@@ -13,15 +13,13 @@ const CheckIn = () => {
   const setResult = (result) => {
     setInfo(checkInUser[result]);
   };
-  const [event, setEvent] = useState(mockEvents[0]);
+  const [event, setEvent] = useState(mockEvents[0].name);
```

Comment:
there should be no default event, it should just be like something generic like "No Event Selected"